src/Artesian/MarketData/_Dto/CheckConversionResults.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+from typing import List
+
+
+@dataclass
+class CheckConversionResults:
+    """
+    Class for the CheckConversionResults.
+
+    Attributes:
+        targetUnitOfMeasure: the target UnitOfMeasure
+        convertibleInputUnitOfMeasure: the list of convertible input UnitOfMeasure
+        notConvertibleInputUnitOfMeasure: the list of not convertible input
+                                          UnitOfMeasure
+    """
+
+    targetUnitOfMeasure: str
+    convertibleInputUnitOfMeasure: List[str]
```

Comment:
there is a mismatch with the parameters.

these are List, should be plural if the convention is to pluralize names.

 check also API for coherence.